dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java

```diff
@@ -192,23 +196,19 @@ public void doSaveProperties(long version) {
 
     private void loadProperties() {
         if (file != null && file.exists()) {
-            InputStream in = null;
-            try {
-                in = new FileInputStream(file);
+            try (InputStream in = new FileInputStream(file)) {
                 properties.load(in);
                 if (logger.isInfoEnabled()) {
-                    logger.info("Load registry store file " + file + ", data: " + properties);
+                    logger.info(
+                        String.format(
+                            "Load registry store file %s, data: %s", file, properties

```

Comment:
I think better to use string concatination, as info log level this might called lot many time and string concatination has better performance the String.format.

Another place where place I could think we can make more use of make logging, if present the file full path instead of file object.